components/shared/indexIssueDetails/IndexIssueDetails.js

```diff
@@ -167,7 +168,7 @@ export default function IndexIssueDetails({
     <>
       <BackLink indexType={indexType} history={router} />
       <h4 className="text-secondary mb-0">
-        {indexType === "project" ? "Project details" : "Product Details"}
+        {indexType === "project" ? "Project Details" : "Product Details"}
```

Comment:
It bothered me that these weren't both capitalized

Reply:
nice catch! there is some tension around here wrt to title casing so i may have been splitting the difference 🤪  @patrickm02L @amenity

Reply:
Capitalize both. There are other examples on the page that makes that capitalization legit.

**_examples_**

<img width="570" alt="Screen Shot 2022-10-28 at 7 41 59 AM" src="https://user-images.githubusercontent.com/5566727/198589407-73e893e8-4e21-4b6f-a3c6-9407b835d197.png">

<img width="462" alt="Screen Shot 2022-10-28 at 7 42 03 AM" src="https://user-images.githubusercontent.com/5566727/198589437-7fc3218d-184f-4744-95f8-9b1e2c9c2c75.png">